types/V1.mli

```diff
@@ -777,19 +777,31 @@ module type STACKV4 = sig
       UDPv4 [port] and immediately return.  If [port] is invalid (not
       between 0 and 65535 inclusive), it raises [Invalid_argument].
       Multiple bindings to the same port will overwrite previous
-      bindings, so callbacks will not chain if ports clash. *)
+      bindings, so callbacks will not chain if ports clash.
+      Note that [listen] must be called again after new callbacks are
+      registered in order to have traffic sent to them.
+      *)
 
   val listen_tcpv4: t -> port:int -> TCPV4.callback -> unit
   (** [listen_tcpv4 t ~port cb] registers the [cb] callback on the
       TCPv4 [port] and immediatey return.  If [port] is invalid (not
       between 0 and 65535 inclusive), it raises [Invalid_argument].
       Multiple bindings to the same port will overwrite previous
-      bindings, so callbacks will not chain if ports clash. *)
+      bindings, so callbacks will not chain if ports clash.
+      Note that [listen] must be called again after new callbacks are
+      registered in order to have traffic sent to them.
+      *)
```

Comment:
Let me make sure I'm right about this in the first place before I attempt to answer that question.